brian2tools/baseexport/collector.py

```diff
@@ -133,22 +158,24 @@ def collect_Events(group):
                                       'when': group.thresholder[event].when,
                                       'order': group.thresholder[event].order,
                                       'dt': group.thresholder[event].clock.dt}
+        event_identifiers |= (get_identifiers(group.events[event]))
```

Comment:
Tiny nitpicking comment: you don't need the outer parentheses on the right-hand-side.